darwin/DarwinProcess.c

```diff
@@ -330,30 +344,34 @@ void DarwinProcess_setFromKInfoProc(Process* proc, const struct kinfo_proc* ps,
    proc->updated = true;
 }
 
-void DarwinProcess_setFromLibprocPidinfo(DarwinProcess* proc, DarwinProcessList* dpl, double time_interval) {
+void DarwinProcess_setFromLibprocPidinfo(DarwinProcess* proc, DarwinProcessList* dpl, double time_interval_ns) {
```

Comment:
I started incorporating units into the names of variables. Given that there's no proper higher-order "Measurement" class/abstraction that manages this, I figure Hungarian notation is the next best thing.

Reply:
Camel-case is the overriding style used for variables and parameters throughout htop, I'd stick with that here too.